_posts/2019-07-04-tip-autoconnect-order.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Automatic Connections to New Nodes"
+subtitle: "Tip of the Day"
+category: tips
+tags: [nodes]
+---
+
+<img width="200" class="float-left" style="margin-right: 20px;" src="{{ site.baseurl }}/resources/images/2019-07-04-tip-autoconnect-order/spaghetti.png" alt="Spaghetti">
+When creating _Switch_, _Group_ or _Parent_ nodes in the _Graph Editor_, Gaffer
```

Comment:
Node names don't need italics.

Reply:
Fixed in https://github.com/GafferHQ/gafferhq.github.io/pull/45/commits/bfcadc29ebb24409c7911b39ec2930efdda31093